apps/molgenis-components/src/components/forms/ButtonDanger.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <button type="button" class="btn btn-danger" @click="onClick">
+    <slot />
+  </button>
+</template>
+
+<script>
+import ButtonAction from "./ButtonAction.vue";
+
+export default {
+  extends: ButtonAction,
+};
+</script>
+
+<docs>
+Example
+
+```jsx
+<template>
```

Comment:
I'm a bit confused, in the big PR ( #1196) all the docs are html, but this one is still markdown. Do we have a preference or doesn't it matter?